src/config/file_lines.rs

```diff
@@ -163,7 +163,7 @@ impl fmt::Display for FileLines {
             Some(map) => {
                 for (file_name, ranges) in map.iter() {
                     write!(f, "{}: ", file_name)?;
-                    write!(f, "{}\n", ranges.iter().format(", "))?;
+                    writeln!(f, "{}", ranges.iter().format(", "))?;
```

Comment:
warning: using `write!()` with a format string that ends in a single newline
```
   --> src\config\file_lines.rs:166:21
    |
166 |                     write!(f, "{}\n", ranges.iter().format(", "))?;
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
```
    = note: `#[warn(clippy::write_with_newline)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#write_with_newline